nemo_text_processing/text_normalization/zh/data/char/lower_to_upper.tsv

```diff
@@ -0,0 +1,26 @@
+a	A
+b	B
+c	C
+d	D
+e	E
+f	F
+g	G
+h	H
+i	I
+j	J
+k	K
+l	L
+m	M
+n	N
+o	O
+p	P
+q	Q
+r	R
+s	S
+t	T
+u	U
+v	V
+w	W
+x	X
+y	Y
+z	Z
```

Comment:
Thanks for your PR. Is there any reason we need this mapping? It seems a trivia op in Python.

Reply:
In this case,we want to process in graph and it could be set as a part of postprocessor instead of applying to string in public normalize.py.I have the plan to add chinese upper and lower forms in this tsv when I got engough tsv data.just like trans "壹“ to ”一“